README.md

```diff
@@ -61,6 +179,23 @@ Binding preference is determined by the relative orientation of the first and la
 | &rarr; | &darr; | G |
 | &rarr; | &larr; | T |
 
+
+### Authors' Note: 
+A number of ambiguities present in the original program are given defined behavior in this rendition of the system, listed below. 
+
+**Binding Preference of Single Amino Acid Enzymes**
```

Comment:
I thought this rule was for single base strands? Not single amino acid enzymes. Unless I'm misunderstanding something. Because if there was an enzyme with one amino acid, its first amino acid is also its last one, so the relative orientation is always the same direction, so it would always bind to A.

Reply:
Yes, the clarification is for single amino acid strands. They do not have behavior as written, unless you allow that the first element in the folding pattern is also the last element. (which is how it is implemented currently). Single base strands do nothing and have no binding preference because they do not encode for any amino acids